adapters/adquery/adquerytest/exemplary/data-null.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Should categories json tests as `supplemental` and `exemplary`. Note that `exemplary` are for the good responses and `supplemental` are for no response error cases